### DISCUSSION TITLE
企业付款到用户零钱  产品权限验证失败,请查看您当前是否具有该产品的权限。无需回复。

### DISCUSSION THREAD

**Discussion by sunkangchina** [w7corp/easywechat#2508] [Q&A] [answered]
https://easywechat.com/5.x/payment/transfer.html

![image](https://user-images.githubusercontent.com/66815206/175201754-f6bea945-6395-4a62-bb57-72b1ed5e9fb1.png)
![image](https://user-images.githubusercontent.com/66815206/175201949-9778959e-0e70-4c6d-86ac-4507aa81913b.png)

对应权限.
![image](https://user-images.githubusercontent.com/66815206/175204053-50ce65ac-4a22-4ecf-9914-8da03c1426cb.png)

 https://pay.weixin.qq.com/index.php/public/product/detail?pid=5&productType=0

**Comment by overtrue** [comment, marked as answer]
问题是啥？

**Reply by sunkangchina** [reply]
企业付款到零钱  与 企业转帐到零钱不是同一个产品。 目前企业付款到零钱应该是已经不存在了，只有企业转帐到零钱！ 5.X没有企业转帐到零钱。

**Reply by sunkangchina** [reply]
5.x 有企业付款到零钱
     **没有企业转帐到零钱**

**Reply by zerowang009** [reply]
我用的4.多的版本，没有这个商家转账到零钱的接口，什么时候更新呢？

**Reply by overtrue** [reply]
4.x 不在新增接口，你可以选择升级到 5.x，唯一的变化就是 oauth 登录那里改了一个方法名

**Reply by zerowang009** [reply]
大佬，能详情点不  我看5.x的文档 ，也没商家转账到零钱的文档啊，还是之前的企业付款到零钱。

**Reply by sunkangchina** [reply]
> 前的企业付款到零钱。
这是两个产品，如果是easywechat6.x的就有V3的接口了，如果是以前老的，那是没有的。
